frontend/app/src/App.js

```diff
@@ -1,25 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+
+import TextField from '@mui/material/TextField';
+
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+
+const Input = createBrowserRouter([
+  {
+    path: "/documents/new",
+    element: <div>
+      <TextField id="outlined-basic" label="Outlined" variant="outlined" />
+    </div>,
+  },
+]);
```

Comment:
Please move this to a separate file and rename it. It's you'r app's main router.
e.g. you can move it to `/frontend/browser-router.js` and name it e.g. browserRouter.
use NamingLikeThis for component files and naming-like-this or namingLikeThis for helper files e.g. index.js, helpers.js, etc..